src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -82,38 +82,10 @@
         flat
         class="section-container mt-4"
       >
-        <v-row no-gutters>
-          <v-col
-            cols="12"
-            sm="3"
-          >
-            <label>Add a Foreign Business</label>
-          </v-col>
-
-          <v-col
-            cols="12"
-            sm="8"
-            class="ml-8"
-          >
-            <span>**TODO**</span>
-
-            <v-row
-              class="justify-end mr-0 mt-2"
-            >
-              <v-btn
-                id="app-cancel-btn"
-                large
-                outlined
-                color="primary"
-                @click="isAddingAmalgamatingForeignBusiness = false"
-              >
-                <span>Cancel</span>
-              </v-btn>
-            </v-row>
-          </v-col>
-
-          <!-- extra column is for possible action button -->
-        </v-row>
+        <ForeignBusiness
+          @add:foreignBusiness="saveAmalgamatingForeignBusiness($event)"
+          @cancel="isAddingAmalgamatingForeignBusiness = false"
+        />
```

Comment:
You've moved some layout code (the label, rows/cols and cancel button) into your component. But now this page has that layout code for the other button, so the layout is in 2 places. Do you recommend moving the other button's code to its own sub-component, or moving your code back in here?

Reply:
I think it depends on how much we want to limit complexity in the parent component. I can move the code into AmalgamatingBusinesses.vue and get rid of ForeignBusiness.vue, and then move the jurisdiction functions into a mixin (like jurisdictionOptions and onJurisdictionChange) to not increase complexity too much.

Reply:
For sure I'd like to limit complexity, but I'd prefer to keep the layout on the same page.

👍 to your edit.